tests/conftest.py

```diff
@@ -11,7 +11,7 @@
 
 # See https://hypothesis.readthedocs.io/en/latest/strategies.html#interaction-with-pytest-cov
 try:
-    import hypothesis  # noqa
+    from hypothesis import given  # noqa
```

Comment:
I changed this because even when uninstalling `hypothesis`, the import is still valid.
Requesting `given` will raise the error